### DISCUSSION TITLE
import package

### DISCUSSION THREAD

**Discussion by sebastianjhm** [pyodide/pyodide#2002] [Q&A]
I need to install two packages from my project. The first is "glpk" that is available in pyodide's packages, and the second is [mip](https://python-mip.readthedocs.io/en/latest/install.html), that not is in pyodide's packages. How can I install this 2 packages?

**Comment by ryanking13** [comment]
`mlp` can be installed through `micopip.install("mlp")`.

For `glpk`, what we now have in Pyodide is glpk library, not [PyGLPK](https://pypi.org/project/glpk/). So if you want to use PyGLPK, it needs to be ported manually.
